test/benchmark/bm_iroha_ed25519.cpp

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <ed25519/ed25519.h>
+
+#include <cstdlib>
+#include <vector>
+
+#include <benchmark/benchmark.h>
+
+auto ConstructRandomVector(int64_t size) {
+  std::vector<unsigned char> v;
+  v.reserve(static_cast<int>(size));
```

Comment:
The cast of int64_t to int looks very suspicious.

Reply:
It should indeed work without it, thanks!